microsoft-edge/webview2/reference/javascript/hostobjectsoptions.yml

```
@@ -96,6 +96,20 @@ properties:
       content: 'log: (...data: any[]) => void;'
       return:
         type: '(...data: any[]) =&gt; void'
+  - name: shouldPassTypedArraysAsArrays
+    uid: 'WebView2Script!HostObjectsOptions#shouldPassTypedArraysAsArrays:member'
+    package: WebView2Script!
+    fullName: shouldPassTypedArraysAsArrays
+    summary: >-
+      By default, typed arrays will be passed to host as IDispatch. Otherwise, set to true to pass typed arrays to host
+      as array.
```

```suggestion
      By default, typed arrays are passed to the host as `IDispatch`. To instead pass typed arrays to the host
      as `array`, set this to `true`.
```
* assuming backticks are supported here.
* generally prefer present tense (instead of future tense) per corp-level style guide - https://learn.microsoft.com/en-us/style-guide/grammar/verbs -- "The present tense is often easier to read and understand than the past or future tense. It’s the best choice for most content."
* favor pattern: To accomplish X, do Y."